docs-src/typescript/how-to-handle-workflow-logic-requirements-in-typescript.md

```diff
@@ -0,0 +1,95 @@
+---
+id: how-to-handle-workflow-logic-requirements-in-typescript
+title: How to handle Workflow logic requirements in TypeScript
+sidebar_label: Workflow logic requirements
+description: Handle Signal
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+In the Temporal TypeScript SDK, Workflows run in a deterministic sandboxed environment.
+The code is bundled on Worker creation using Webpack, and any package can be imported as long as it does not reference Node.js or DOM APIs.
```

Comment:
```suggestion
The code is bundled on Worker creation using Webpack, and can import any package as long as it does not reference Node.js or DOM APIs.
```

Reply:
I'd consider mentioning how to work around these limitations if you have a library that references those APIs and you're sure that those are not used at runtime.

https://typescript.temporal.io/api/interfaces/worker.BundleOptions#ignoremodules